### PR TITLE
Fix/event types list nickname

### PR DESCRIPTION
**Pull request by dmitriy-uvin**

### CODE REVIEW

File: client/src/components/event-types/all-event-types/ActionBlock.vue [hunk 1]
```diff
@@ -8,7 +8,9 @@
                 <VCol cols="10">
                     <span class="user-name">{{ user.name }}</span>
                     <br />
-                    <a href="">shedulia.xyz/nickname</a>
+                    <RouterLink :to="{ path: user.nickname }">
+                        shedulia.xyz/{{ user.nickname }}
```
Comment (yevhen-turovets) [inline client/src/components/event-types/all-event-types/ActionBlock.vue:12]: Add to language file `shedulia.xyz`

Comment (dmitriy-uvin) [reply client/src/components/event-types/all-event-types/ActionBlock.vue:12]: How it will be translated?

Comment (yevhen-turovets) [reply client/src/components/event-types/all-event-types/ActionBlock.vue:12]: No, they not translate. I said wrong, you just need to output it as a constant.  I suggest adding to .env

**Review by aLLeXUs — CHANGES_REQUESTED** [review]
We have `VUE_APP_API_URL` variable. You can use it for getting a domain name.

**Review by aLLeXUs — APPROVED** [review]

**Pull request merged**